core/src/components/popover/animations/ios.enter.ts

```diff
@@ -122,20 +109,8 @@ export const iosEnterAnimation = (baseEl: HTMLElement, opts?: any): Animation =>
         contentEl.style.setProperty('bottom', `${bottom}px`);
       }
 
-      const safeAreaLeft = ' + var(--ion-safe-area-left, 0)';
-      const safeAreaRight = ' - var(--ion-safe-area-right, 0)';
-
-      let leftValue = `${left}px`;
-
-      if (checkSafeAreaLeft) {
-        leftValue = `${left}px${safeAreaLeft}`;
-      }
-      if (checkSafeAreaRight) {
-        leftValue = `${left}px${safeAreaRight}`;
-      }
```

Comment:
While removing these looks like a step backwards at a glance, they weren't actually working as intended because the `checkSafeAreaLeft/Right` variables were based on the hardcoded 0/25px guess at the margins. I've removed them to ensure the MD and iOS safe area behavior lines up as closely as possible.

Reply:
So I understand: Since we are approximating the safe area bounds we no longer need these checks since it's already handled elsewhere?

Reply:
Right; the adjustment is all done in `calculateWindowAdjustment` now.